@utahdts/utah-design-system-header/src/@types/jsDocTypes.d.js

```diff
@@ -268,6 +268,8 @@
  * @typedef FooterSettings {
  *  @property {string | null | undefined} [copyrightYear] - the year to show next to the copy right symbol in the footer (default is none)
  *  @property {DomLocationTarget} [domLocationTarget] - where in the DOM should the footer be inserted? (defaults to the bottom of the body)
+ *  @property {string | null | undefined } [linkPrivacyPolicy] - a custom link to a privacy policy
```

Comment:
`| undefined` and [linkPrivacyPolicy] both mean undefined. is this intentional? 
[linkPrivacyPolicy] means it does not need specified
`| undefined` means you can pass in undefined
`| undefined [linkPrivacyPolicy]` means it doesn't need passed in and if you do pass it in you can set it to undefined.
it's tempting to suggest just having `| null` and not `| undefined`.

Reply:
change to: `{string | null}`